src/dotnet/commands/dotnet-new/CSharp_Console/$projectName$.csproj

```diff
@@ -12,14 +12,11 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NETCore.App">
-      <Version>1.0.1</Version>
-    </PackageReference>
-    <PackageReference Include="Microsoft.NET.Sdk">
-      <Version>1.0.0-alpha-20161104-2</Version>
+    <PackageReference Include="Microsoft.NETCore.App" Version="1.0.1" />
+    <PackageReference Include="Microsoft.NET.Sdk" Version="1.0.0-alpha-20161104-2">
       <PrivateAssets>All</PrivateAssets>
```

Comment:
This packagereference to the SDK is going to go away in a subsequent change. As a general comment though - NuGet had said they still prefer privateassets\includeassets\excludeassets as elements instead of attributes. @rrelyea